modules/bench/README.adoc

```diff
@@ -6,6 +6,14 @@ Auctionmark runs in some capacity currently, this guide describes the requiremen
 - Run AWS_PROFILE=<profile_name_containing_above_creds> bin/download-dataset.sh --auctionmark (skip)
 - See comments at bottom of xtdb.bench2.xtdb.clj for further instructions.
 
+=== Creating a C1 vs C2 comparison report
+
+The following setup assumes that you have `2.x` checked out and another directory at `../xtdb` which can checkout the `auctionmark` branch of C1. Make sure there is nothing to be stashed on your (presumably C1) working branch.
```

Comment:
nit: with AsciiDoc, the [best practice](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line) is one sentence per line.